pkg/controller/controlplane/valuesprovider.go

```diff
@@ -648,6 +696,22 @@ func getAuthNGroupRoleChartValues(cpConfig *apismetal.ControlPlaneConfig, cluste
 	return values, nil
 }
 
+// returns values for "auditlog-webhook"
+func getAuditlogChartValues(cpConfig *apismetal.ControlPlaneConfig, cluster *extensionscontroller.Cluster, config config.Audit) (map[string]interface{}, error) {
+
+	values := map[string]interface{}{
+		"auditlogWebhook": map[string]interface{}{
+			"enabled": config.Enabled,
+			"splunk-hec-endpoint": map[string]interface{}{
+				"url":   "https://splunk.cns.cloud.fi-ts.io/",
+				"token": "fits_in_hec-cloud-native_TEST_34BC91E8-DFAC-E799-73B9-BF5B8A40611B",
```

Comment:
Oh. Did you expose the token? :D 
Just a placeholder would possibly be a little nicer until you can retrieve those values from the masterdata-api.

Reply:
Oh no! D:
Changed to placeholders; requested revocation of that test token.